core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -330,6 +330,28 @@ private static String toBase64_(byte[] bytes) {
     }
   }
 
+  /** SQL BASE64(string) function for string. */
+  public static String base64(String string) {
```

Comment:
There already exists `toBase64(ByteString string)` and `toBase64(String string)` in SqlFunctions.

Reply:
You mean here reuse toBase64 method?  It is OK for it.

Reply:
Yes, it would be much better if we can reuse or add arguments to existing methods.

Reply:
Done